### PR TITLE
fix (api, ui): relaunch button

### PR DESCRIPTION
**Pull request by yesnault**
take care of pipeline parameters & payload (relaunch workflow)
take care of pipeline paremters (re-play pipeline)
delete unused dropdown from breadcrumb

Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

**Comment by yesnault** [comment]
@fsamin Right
We need "HookEvent", "Manual" and only "PipelineParameters"
buildParameters is now with DisableDetailledNodeRun=false
It's not only for the root pipeline, this fix include relaunch of child pipelines (for pipeline_parameters)

File: ui/src/app/shared/workflow/node/run/node.run.param.component.ts [hunk 1]
```diff
@@ -122,13 +122,27 @@ export class WorkflowNodeRunParamComponent {
...
         if (this.workflowRun && this.workflowRun.workflow.root_id !== this.nodeToRun.id) {
             this.readOnly = true;
             this.codeMirrorConfig = Object.assign({}, this.codeMirrorConfig, {readOnly: true});
         } else {
             this.readOnly = false;
+            isRoot = true;
             this.codeMirrorConfig = Object.assign({}, this.codeMirrorConfig, {readOnly: false});
         }
+
+        if (isRoot === true && this.workflowRun) {
```
Comment (sguiheux) [inline ui/src/app/shared/workflow/node/run/node.run.param.component.ts]: remove  ->  === true

**Comment by fsamin** [comment]
@yesnault 
and what about calling `getWorkflowNodeRunHandler` to get the detail on the node run when the model is opening ? as we do when we doubleclick on it.

So we could lighten `getWorkflowRunHandler` event more

**Comment by sguiheux** [comment]
MIssing readonly mode on list parameter if we relaunch a noderun ?

**Pull request merged**